.github/ISSUE_TEMPLATE/bug-en.md

```diff
@@ -0,0 +1,91 @@
+---
+name: Bug Report
+about: Create a bug report of Xray.
+title: '[Bug] <bug you are reporting>'
+labels: ''
+assignees: ''
+
+---
+
+<!-- Thanks for your reporting.
+1. This issue is used to submit a bug of Xray. If you are not sure, please give feedback in the group.
+2. Please check the existing issues, discussions and documentation in detail. Issue which report a known bug may be closed.
```

Comment:
```suggestion
2. Please check existing issues and discussions first and read the documentation in detail. If you are submitting a known issue, it will be closed
```